actions.go

```diff
@@ -100,6 +102,7 @@ type CustomActionsTriggerDefinitionUpdateInput struct {
 	Published              *bool                                           `json:"published,omitempty"`
 	AccessControl          CustomActionsTriggerDefinitionAccessControlEnum `json:"accessControl,omitempty"`
 	ResponseTemplate       *string                                         `json:"responseTemplate,omitempty"`
+	EntityType             CustomActionsEntityTypeEnum                     `json:"entityType,omitempty"`
```

Comment:
we might need to add an test to ensure we can "reset" this back to the default value.  I cannot remember if this is a terraform problem or an opslevel-go problem.